features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/SwitchCompatMapperTest.kt

```diff
@@ -39,40 +54,23 @@ internal class SwitchCompatMapperTest : BaseSwitchCompatMapperTest() {
         )
     }
 
-    @Test
-    fun `M resolve the switch as wireframes W map() { checked }`() {
+    @RepeatedTest(8)
```

Comment:
why are we repeating this test?

Reply:
because in the function forge generate random data, so I want to repeat the test so that  it can cover maximum the case